tests/correction_test.py

```diff
@@ -240,3 +241,26 @@ def test_polarization_analysis_workflow_creation(
             pol.supermirror.SupermirrorEfficiencyFunction[elem] in graph.keys()
         ) == (wf is _supermirror_workflow)
         assert (pol.he3.He3CellPressure[elem] in graph.keys()) == (wf is _he3_workflow)
+
+
+@pytest.mark.parametrize('f', [0.1, 0.5, 0.9, 0.99, 1.0])
+def test_half_polarized_with_flipper_computes_and_applies_matrix_inverse(
+    f: float,
+) -> None:
+    ground_truth = np.array([7.0, 11.0])
+    polarizer = np.array([[1.1, 0.7], [0.7, 1.1]])
+    flipper = np.array([[1.0, 0.0], [1 - f, f]])
+    intensity = flipper @ polarizer @ ground_truth
+
+    workflow = CorrectionWorkflow(half_polarized=True)
+    workflow[TransmissionFunction[Polarizer]] = FakeTransmissionFunction(polarizer)
+    workflow[ReducedSampleDataBySpinChannel[Up, pol.NoAnalyzer]] = intensity[0]
+    workflow[ReducedSampleDataBySpinChannel[Down, pol.NoAnalyzer]] = intensity[1]
+    workflow[FlipperEfficiency[Polarizer]] = FlipperEfficiency(f)
```

Comment:
@astellhorn Have a look at these lines, they should answer your comments above: Here I used `NoAnalyzer` as a value for the "analyzer spin", but this is just an internal alias for "Up" (arbitrary choice).

Reply:
Yes I understand

Reply:
For testing the polarizer: useful values to test would be either (i)  for a supermirror T_Up + T_Down = 1 (e.g. 0.96 and 0.04), or for (ii) a 3He cell T_Up + T_Down < 1 (1 only for perfect transmisison through the cell), (e.g. 0.8 and 0.05).

Reply:
I am sorry I still somehow cant test this myself as when compiling the `correction_test.py`, I still get the error message `cannot import name 'assert_allclose' from 'scipp.testing'`

Reply:
You may need to update your Scipp version.

Reply:
> For testing the polarizer: useful values to test would be either (i) for a supermirror T_Up + T_Down = 1 (e.g. 0.96 and 0.04), or for (ii) a 3He cell T_Up + T_Down < 1 (1 only for perfect transmisison through the cell), (e.g. 0.8 and 0.05).

@astellhorn Can you clarify which values you mean? `f`? It is being tested for a range of values already.

Reply:
I meant the polarizer matrix values. Currently they are `polarizer = np.array([[1.1, 0.7], [0.7, 1.1]])`.

Reply:
Ok, I updated it. IT does not actually matter for this test, as long as the matrix structure is right and it has an inverse.

Reply:
Thanks! Yes, that is true. It was more for clarity.